Update docs on muting CSM issues (#26614)
DataDog/documentation: content/en/security/cloud_security_management/review_remediate/mute_issues.md

**Note**: Muting a misconfiguration removes it from the calculation of your posture score.

{{< img src="security/csm/mute_issue-2.png" alt="The Mute Issue dialog box contains fields for specifying the reason and duration of the mute" style="width:100%;">}}

1. On the misconfiguration, vulnerability, or identity risk side panel, find the triage status dropdown for the resource.
   - If the side panel is for a single resource, under **Next Steps**, the list is under **Triage**.
   - If the side panel is for a list of aggregated resources, select one or more resources in the table, then find the list in the **Status** column.

> 2. Open the list with the current triage status and click **Muted**. The Mute issue window opens.

Reviewer: ```suggestion
2. Open the list with the current triage status and click **Muted**. The **Mute issue** window opens.
```

3. Select a reason for the mute; for example, a fix is pending, it's a false positive, or it's an accepted risk.
4. Enter an optional **Description**.
5. Select the duration of the mute.

> 6. Click **Mute**. The Mute issue window closes.

Reviewer: ```suggestion
6. Click **Mute**. The **Mute issue** window closes.
```

### Unmute an issue

Muted issues automatically unmute after the specified mute duration expires. You can also manually unmute an issue.

1. On the misconfiguration, vulnerability, or identity risk side panel, find the triage status dropdown for the resource.
   - If the side panel is for a single resource, under **Next Steps**, the list is under **Triage**.
   - If the side panel is for a list of aggregated resources, select one or more resources in the table, then find the list in the **Status** column.

> 2. Click **Muted** to open the dropdown, then select a new triage status. The triage status updates immediately for the selected resource(s).

Reviewer: ```suggestion
2. Click **Muted** to open the dropdown, then select a new triage status. The triage status updates immediately for the selected resources.
```

(IIRC when people are using machine translation, usage of (s) can mess it up.)
